src/views/Checkout.vue

```diff
@@ -51,7 +51,7 @@
 
         <button class="global-close nq-button-s" @click="close">
             <ArrowLeftSmallIcon/>
-            Cancel <span>Payment</span>
+            {{ $t('Cancel') }} <span>{{ $t('Payment') }}</span>
```

Comment:
Use [vue-i18n's component interpolation](https://kazupon.github.io/vue-i18n/guide/interpolation.html#slots-syntax-usage) to support languages that need to switch the order of words.

Reply:
One question: Did you have to add "Cancel {payment}" manually to the translation file or did the extractor find it automatically?

And currently, the generated output does not have the desired effect which is hiding just the "payment" part on small mobile screens. Instead the whole element is hidden.

Reply:
Fixed it by adding [component interpolation without a root component as a feature/fix to `vue-i18n`](https://github.com/kazupon/vue-i18n/pull/878).